arquillian-ape-nosql/infinispan-ftest/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <groupId>org.arquillian.ape</groupId>
+    <artifactId>arquillian-ape-nosql-parent</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-ape-nosql-infinispan-ftest</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Arquillian Ape NoSQL Infinispan Functional Test</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.lordofthejars</groupId>
+      <artifactId>nosqlunit-infinispan</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.arquillian.cube</groupId>
+      <artifactId>arquillian-cube-docker</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.arquillian.ape</groupId>
+      <artifactId>arquillian-ape-nosql-infinispan</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.arquillian.junit</groupId>
+      <artifactId>arquillian-junit-standalone</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
```

Comment:
I know we have both `assertj` and `junit` defined as `test` scope in the parent pom's `dependencyManagement`. The problem here is however that this information is not easily available and thus making it confusing here. All other dependencies have explicit `test` scope specified but these not - then we can start wondering why is it so, as default scope when not specified is not `test`. Can we agree on a convention here?